_posts/2024-11-29-quarkus-jlama.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Creating pure Java LLM infused application with Quarkus, Langchain4j and Jlama"
+date: 2024-11-29
```

Comment:
@mariofusco can you update the date and then we are good to go.

Without it may appear behind some other posts.